dChatServer/PlayerContainer.h

```diff
@@ -42,7 +43,8 @@ struct PlayerData {
 	std::string playerName;
 	std::vector<FriendData> friends;
 	std::vector<IgnoreData> ignoredPlayers;
-	uint8_t GMLevel = 0;
+	// Unused TODO: make use of this
+	eGameMasterLevel GMLevel = eGameMasterLevel::CIVILIAN;
```

Comment:
I would remove the include, forward declare and move the initializer to be defined in PlayerContainer.cpp to reduce the include clutter